pkg/gofr/queryparam/queryparams.go

```diff
@@ -0,0 +1,7 @@
+package queryparam
```

Comment:
why is queryParam created as a new package? I think it could be a part of gofr package

Reply:
**queryParam** is created as a new package because I was facing **cyclic dependency** with **cmd** package if i keep that interface inside **gofr** package